index.html

```diff
@@ -136,6 +136,7 @@
         <pre class="example highlight json" title="typical manifest">
 {
   "lang": "en",
+  "dir": "ltr",
```

Comment:
Why was dir chosen instead of direction or directionality? Is this a common abbreviation for this? It made me think of directory :)